### ISSUE TITLE
Fix issues from Pagination visual, color, and color contrast review

### ISSUE THREAD

**Issue by rsmithadhoc** [department-of-veterans-affairs/vets-design-system-documentation#2112]
## Description

Correct issues with `va-pagination` found during visual, color and color contrast accessibility review. 

## Details

- **Consider:** The ellipsis uses three periods ". . ." instead of the ellipsis character "…".
    - <img width="400" alt="image" src="https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/100248909/5b399108-ee14-49a8-b831-93bc4b9a0907">
    - **Recommendation:** Use the ellipsis character. This could also help with screen readers to avoid reading out "period period period".
- **Should:** For the *Previous* and *Next* links, they are only distinguished as links by their color.
    - <img width="400" alt="image" src="https://github.com/department-of-veterans-affairs/vets-design-system-documentation/assets/100248909/8803511e-0141-4a5d-bdd1-9a3ace03b263">
    - **Recommendation:** The source code looks like there should be an icon showing. We should either get that to show or work with design and consider adding an underline to these links.

## Tasks
- [ ] Fix the accessibility issues

## Acceptance Criteria
- [ ] Accessibility issues addressed

**Comment by caw310** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Platform-Design-System-5f8de67192551b0012ebb802/issues/department-of-veterans-affairs/vets-design-system-documentation/2112?planning-poker) with Zenhub @Andrew565 @ataker @harshil1793 @it-harrison @jamigibbs @micahchiang @nickjg231 @powellkerry @rmessina1010 @rsmithadhoc

**Comment by rsmithadhoc** [comment]
https://github.com/department-of-veterans-affairs/va.gov-team/issues/66068 and https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1977 may fix the chevron issue.

**Comment by powellkerry** [comment]
I have confirmed that the icon not showing up in prod is because it is trying to load it from a different directory where the sprite.svg file does not exist. This is being looked at in this issue: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1977
